lightly/models/utils.py

```diff
@@ -463,28 +463,33 @@ def random_token_mask(
 
     return idx_keep, idx_mask
 
-def schedule_momentum(
-    iteration: int, 
-    max_iter: int, 
-    m: float = 0.99
+def cosine_decay_schedule(
+    step: int, 
```

Comment:
Can we just call it `cosine_schedule`? Decay implies that the value decreases but it also supports increasing values.